packages/components/src/Menu/Menu.css

```diff
@@ -8,27 +8,53 @@
 }
 
 .menu {
-  position: absolute;
+  position: fixed;
   right: 0;
+  bottom: 0;
+  left: 0;
   z-index: var(--elevation-menu);
-  width: calc(var(--base-unit) * 12.5);
-  max-height: 300px;
-  box-shadow: var(--shadow-base);
+  width: 100%;
+  max-height: 72vh;
   padding: var(--space-small);
+  padding-bottom: env(safe-area-inset-bottom);
   border: var(--border-base) solid var(--color-grey--lighter);
-  border-radius: var(--radius-base);
-  overflow: auto;
+  border-radius: var(--radius-base) var(--radius-base) 0 0;
+  overflow-y: scroll;
   background-color: var(--color-white);
+  -webkit-overflow-scrolling: touch;
 }
 
-.above {
-  bottom: 100%;
-  margin-bottom: var(--space-smallest);
+@media screen and (max-width: 639px), print {
+  .menu:last-child {
+    padding-bottom: var(--space-small);
+  }
 }
 
-.below {
-  top: 100%;
-  margin-top: var(--space-smallest);
+@media only screen and (min-width: 640px), print {
+  .menu {
```

Comment:
a few attributes here are double declared from the small version and can be removed.

```
z-index
border
background-color
```

Reply:
✅